chain/dev/config.toml

```diff
@@ -18,7 +18,7 @@ digest = ""
 genesis = "./chain/dev/genesis-spec.json"
 
 [account]
-key = "alice"
+key = ""
```

Comment:
Are we sure we don't want a key anymore?

Reply:
pretty sure? We usually specify the key when starting a node anyways.